runtime/platform/compiler.h

```diff
@@ -99,15 +99,8 @@
 
 #endif // (__cplusplus) >= 202002L
 
-/// Define a C symbol with weak linkage.
-#ifdef _MSC_VER
-// There currently doesn't seem to be a great way to do this in Windows and
-// given that weak linkage is not really critical on Windows, we'll just leave
-// it as a stub.
-#define ET_WEAK
```

Comment:
> @vortex-captain does this work with MSVC now? Is there a certain version of MSVC this works for, because i remember it didn't work for the older versions.

yes, Windows x64 build uses clang-cl compiler, which supports `__attribute__((weak))`